force-app/main/default/lwc/.eslintrc.json

```
@@ -1,3 +1,6 @@
 {
-  "extends": ["@salesforce/eslint-config-lwc/recommended"]
+  "extends": [
+    "@salesforce/eslint-config-lwc/recommended",
+    "@locker/eslint-config-locker"
```

**question:** why using the basic one ?

And why not using the [recommended one](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.security_lwsec_tools_lint#:~:text=If%20you%20want%20notification) ?

---

Maybe we could also add the recommended one for locker ?

```suggestion
    "@locker/eslint-config-locker/recommended"
```

---

Ah yes! in fact I missunderstood your first question. New commit is coming, thanks mate !